test/controls/nav-link.test.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react'
+import { mount } from 'enzyme'
+import { NavLink } from '../../src/'
+
+test('NavLink renders a link tag with props', () => {
+  const wrapper = mount(<NavLink to="/">Home</NavLink>)
+  expect(wrapper.find('a').exists()).toBe(true)
+  expect(wrapper.props().to).toEqual('/')
```

Comment:
Can you add a check for `activeClassName` being passed down?
You should be able to do `wrapper.find(NavLink)` to get access to the inner component.